README.md

```diff
@@ -107,7 +107,7 @@ space.subscribe('update', (spaceState) => {
 });
 
 // Enter a space, publishing an update event, including optional profile data
-space.enter({
+await space.enter({
```

Comment:
I've added these everywhere in docs to make it clear these are async operations and can be waited upon